model-setup-examples/basic-elevon.md

```diff
@@ -0,0 +1,22 @@
+## Basic Elevon
+
+Elevons are control surfaces that combine the functions of the elevator and the aileron.
```

Comment:
I would like to have here an idea what the example is trying to accomplish. Like a layman way of describing what should happen, e.g. when moving elevator both flaps should move etc. E.g. describe what this programming does and then explain how this is accomplish. E.g. also explain why expotential input is applied.